buildpipeline/pipeline.json

```diff
@@ -107,7 +107,7 @@
           "Name": "DotNet-CoreFx-Trusted-Windows",
           "Parameters": {
             "PB_Platform": "arm",
-            "PB_BuildArguments": "-buildArch=arm -Release -- /p:SignType=real /p:RuntimeOS=win10",
+            "PB_BuildArguments": "-buildArch=arm -Release -- /p:SignType=$(PB_SignType) /p:RuntimeOS=win10",
```

Comment:
I was hoping to say that `/p:SignType=$(PB_SignType)` can be moved to the `PB_PipelineBuildMSBuildArguments`, but I see that var's not used in `...-Windows-NoTests.json`. 😕 Not worth changing in this PR, since you've already done the work and tested it.